.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,27 @@
+name: Docker notebook tests
+
+on:
+  push:
+    branches: [ main ]
```

Comment:
ahh, thanks

Reply:
> I think running this on pushes to main will be a noop as we use PRs rather than directly committing to main...

In my experience, a workflow that triggers on a push to main would in that case run each time a PR is merged to main.

Reply:
@garrison I think you're right, but I think I actually prefer it as a pull request to main. I really hate finding out I broke something after it is in main. We'll keep an eye on the CI budget and make sure all these `apt-gets` aren't killing us, but I think we can leave it on pull requests to main